runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -34,7 +34,7 @@ evaluationDependsOn(":model:fn-execution")
 
 processResources {
   filter org.apache.tools.ant.filters.ReplaceTokens, tokens: [
-    'dataflow.legacy_environment_major_version' : '6',
+    'dataflow.legacy_environment_major_version' : '7',
     'dataflow.fnapi_environment_major_version' : '1',
```

Comment:
Please also update the fnapi version to 7 throughout.

Reply:
Thanks Henning. I'd prefer to change the fniapi separately, as that's an orthogonal change. Moreover, I'm not even sure it's safe (Does fnapi support major versions all the way up to 7?)

Reply:
Dataflow FnAPI supports major version 7, but it's fine to change separately.